waffle_configs/bedrock-prod.env

```diff
@@ -33,6 +33,7 @@ SWITCH_FIREFOX_NEW_COMPARE_OPERA_EXPERIMENT=on
 SWITCH_FIREFOX_UPDATE_NOTIFICATION=on
 SWITCH_FIREFOX_UPDATE_NOTIFICATION_MODAL=on
 SWITCH_FIREFOX_YANDEX=on
+SWITCH_PRE_DOWNLOAD_NEWSLETTER=on
```

Comment:
Should be `SWITCH_FIREFOX_PRE_DOWNLOAD_NEWSLETTER` in both places.

Reply:
whoops, fixing now